substitutions.py

```diff
@@ -0,0 +1,166 @@
+              (r"\$\^{?(\d+)}?\$", r" to the power of \g<1> "), \
+    		  ]        
+
+    symbols = [(r"\\hbar", " hbar "), \
+                (r"\\nabla", " nabla "), \
+                (r"\\infty", " infinity "), \
+                (r"\$?\\sim\$?", " is of order of "), \
+                (r"\\propto", r" proportional to "), \
+                (r"\\neq", r" not equal to "), \
+                (r"\\geq", r" greater than or equal to "), \
+                (r"\\leq", r" less than or equal to "), \
+                (r"\\equiv", r" equivalent to "), \
+                (r"\\approx", r" approximately "), \
+                (r"\\simeq", r" approximately equal to "), \
+                (r" ?\> ?", r" greater than "), \
+                (r" ?\< ?", r" less than "), \
+                (r"\\times", r" times "), \
+                (r"\\pm", r" plus minus "), \
+                (r"\\mp", r" minus plus "), \
+                (r"\\%", r" percent "), \
+                (r"\\dot{(\w*)}", r" \g<1> dot "), \
+                ]
+
+    trig = [(r"\\sin", " sine ", r"\\cos", " cosine "), \
+            (r"\\arcsin", " arc sine "), \
+            (r"\\arccos", " arc cosine "), \
+            ]
+
+    greek = [(r"\\aplha", r" aplha "), \
+             (r"\\[b]eta", r" beta "), \
+             (r"\\[Gg]amma", r" gamma "), \
+             (r"\\[Dd]elta", r" delta "), \
+             (r"\\[var]?epsilon", r" epsilon "), \
+             (r"\\zeta", r" zeta "), \
+             (r"\\eta", r" eta "), \
+             (r"\\[var]?[Tt]heta", r" theta "), \
+             (r"\\iota", r" iota "), \
+             (r"\\[var]?kappa", r" kappa "), \
+             (r"\\[Ll]ambda", r" lambda "), \
+             (r"\\mu", r" mu "), \
+             (r"\\nu", r" nu "), \
+             (r"\\[Xx]i", r" xi "), \
+             (r"\\[var]?[Pp]i", r" pi "), \
+             (r"\\[var]?rho", r" rho "), \
+             (r"\\[var]?[Ss]igma", r" sigma "), \
+             (r"\\tau", r" tau "), \
+             (r"\\[Uu]psilon", r" upsilon "), \
+             (r"\\[var]?[Pp]hi", r" phi "), \
+             (r"\\chi", r" chi "), \
+             (r"\\[Pp]si", r" psi "), \
+             ]
+
+    solarsys = [(r"M ?\$?\_{?\\oplus}?\$?", r" Earth masses "), \
+               (r"M ?\$?\_{?\\odot}?\$?", r" Solar masses "), \
+               (r"M ?\$?\_{?\\star}?\$?", r" Stellar masses "), \
+               (r"R ?\$?\_{?\\oplus}?\$?", r" Earth radii "), \
+               (r"R ?\$?\_{?\\odot}?\$?", r" Solar radii "), \
+               (r"R ?\$?\_{?\\star}?\$?", r" Stellar radii "), \
+               (r"\$?M\$?\_{Jup}\$?", r" Jupiter masses "), \
+               (r"R\$?\_{Jup}\$?", r" Jupiter radii "), \
+               (r"AU", r"AU"), \
+               (r"[^MR]\_\\oplus", r" Earth "), \
+               (r"[^MR]\_\\odot", r" Sun "), \
+               (r"[^MR]\_\\star", r" Star "), \
+               ]
+
+    units = [(r"( .*[^A-Za-z])pc[^A-Za-z]", r"\g<1> parsec "), \
```

Comment:
the issue here is that you might get rid of valid character (e.g. parentheses) just after pc:

```
In [5]: regex_substitute('roughly 3pc)')
Out[5]: 'roughly 3 parsec '
```

(this got rid of the parenthesis)

Do you have any suggestions on how we could improve the regexes to fix that?